gendiff/scripts/args_parser.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import Any
+
+import yaml
+from yaml.loader import SafeLoader
+
+
+def check_format(file_name: str) -> str:
+    return file_name[-4:]
+
+
+def parse_file(file_name: str) -> Any:
```

Comment:
`args_parser` is not a suitable name for what this module is doing. consider renaming to `file_operations` or smth and moving this module out of scripts package as it's not a script